batch/batch/driver/templates/user_resources.html

```diff
@@ -2,16 +2,16 @@
 {% block title %}Batch User Resources{% endblock %}
 {% block content %}
 
-<h1>Users</h1>
+<h1>User Resources</h1>
 <table class="data-table" id="user_resources">
     <thead>
     <tr>
         <th>Name</th>
         <th>Ready Jobs</th>
         <th>Running Jobs</th>
```

Comment:
Can you put ready jobs and cores first, then allocated, then running jobs and cores?